kratos/geometries/geometry.h

```diff
@@ -4102,6 +4083,18 @@ class Geometry
     ///@name Protected Inquiry
     ///@{
 
+    /**
+     * @brief Checks if the geometry points are valid
+     * Checks if the geometry points are valid from the pointer value
+     * Points are not valid when the pointer value is null
+     * @return true All points are valid
+     * @return false At least one point has nullptr value
+     */
+    bool AllPointsAreValid() const
```

Comment:
BTW covering this with a test would be amazing :eyes:

Reply:
I tried, but I didn't come up with a simple solution since it is `protected`...